.github/workflows/target-test.yml

```diff
@@ -90,25 +89,45 @@ jobs:
       matrix:
         idf_ver: ["latest"]
         idf_target: ["esp32", "esp32s2"]
-        example: ["asio_chat", "async_request", "socks4", "ssl_client_server", "tcp_echo_server", "udp_echo_server"]
-
+        example: ["asio_chat", "tcp_echo_server", "udp_echo_server", "ssl_client_server"]
```

Comment:
Question: could we still build the other examples? Even if there's no actual test, it's good I think to have it at least the build step for all of them.